plugins/BEdita/Core/src/Model/Entity/AuthProvider.php

```diff
@@ -61,7 +66,7 @@ protected function _getSlug()
      */
     public function getRoles()
     {
-        $roles = (array)Configure::read(sprintf('Roles.%s', $this->name));
+        $roles = (array)Configure::read(sprintf('Roles.%s', $this->auth_class));
```

Comment:
Are we sure we wish to set allowed roles on an auth class basis? Wouldn't it be possible to give Facebook users the role "facebook" and Twitter users the role "twitter" while using the same underlying ~~authentication~~ authorization mechanism?

Reply:
Don't know. Leaved here because it's used with UUID I guess.
Usually user roles do not depend on auth provider used, I'd prefer some info on external auth providers used in `GET /users/{id}` for instance. But generally not a role.